DependencyInjection/EkinoNewRelicExtension.php

```diff
@@ -120,7 +120,7 @@ public function load(array $configs, ContainerBuilder $container): void
             $loader->load('twig.xml');
         }
 
-        if ($config['enabled'] && $config['monolog']['enabled']) {
+        if (\extension_loaded('newrelic') && $config['enabled'] && $config['monolog']['enabled']) {
```

Comment:
This is done in Configuration.php already, isnt it?
Or that is only if you use: `enable: auto`?

Reply:
enabled needs to be a boolean or you get this error:
```
Invalid type for path "ekino_new_relic.enabled". Expected boolean, but got string.
```

Reply:
When the value is set to true, but the newrelic bundle is not installed you get this errors:
```
The newrelic PHP extension is required to use the NewRelicHandler
```

Which is thrown within the Monolog NewRelicHandler:
```
    protected function write(array $record)
    {
        if (!$this->isNewRelicEnabled()) {
            throw new MissingExtensionException('The newrelic PHP extension is required to use the NewRelicHandler');
        }
```

Reply:
Sounds like an expected behavior. If you enable it, but don't have the extension, it'll faill.

Like using doctrine with pgsql DSN without the extension.

Reply:
That's a fair point, I guess the issue is having the same yaml file for machines with the extension installed and then local machines without it.

What I should do it use an `.env`  value to default to false locally and then true on deployed installs.